ra-optimisation/linear-rao/src/test/resources/LinearRaoParameters.json

```diff
@@ -1,5 +1,7 @@
 {
   "version" : "1.0",
+  "dc-mode" : false,
+  "ac-to-dc-fallback" : false,
```

Comment:
is it useful ?

Reply:
This file is the expected result of the round trip test, and I guess we want to export these parameters, regardless if they are explicitly specified or not.